### ISSUE TITLE
npm install @apollo/gateway fails due to  @apollo/query-planner-wasm failure

### ISSUE THREAD

**Issue by sce9sc** [apollographql/federation#184]
There are 2 issues. 

1) It needs wasm-pack which needs Rust
I Installed rust using rustup‑init.exe since I am in windows 10 environment 
and also downloaded and run wasm-pack-init.exe
then run npm i @apollo/gateway
2) Failed at the @apollo/query-planner-wasm@0.0.5 preinstall script.
Output below

```

npm i @apollo/gateway

> @apollo/query-planner-wasm@0.0.5 preinstall C:\Users\gzlabs\Documents\work\Projects\apollo-gateway\node_modules\@apollo\query-planner-wasm
> npm run wasm-pack


> @apollo/query-planner-wasm@0.0.5 wasm-pack C:\Users\gzlabs\Documents\work\Projects\apollo-gateway\node_modules\@apollo\query-planner-wasm
> wasm-pack build --target nodejs --out-dir dist --out-name index --scope apollo

Error: crate directory is missing a `Cargo.toml` file; is `.` the wrong directory?
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @apollo/query-planner-wasm@0.0.5 wasm-pack: `wasm-pack build --target nodejs --out-dir dist --out-name index --scope apollo`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @apollo/query-planner-wasm@0.0.5 wasm-pack script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\gzlabs\AppData\Roaming\npm-cache\_logs\2020-09-25T09_38_43_194Z-debug.log
npm WARN graphql-subscriptions@1.1.0 requires a peer of graphql@^0.10.5 || ^0.11.3 || ^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-upload@8.1.0 requires a peer of graphql@0.13.1 - 14 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-gateway@1.0.0 No description
npm WARN apollo-gateway@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @apollo/query-planner-wasm@0.0.5 preinstall: `npm run wasm-pack`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @apollo/query-planner-wasm@0.0.5 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\gzlabs\AppData\Roaming\npm-cache\_logs\2020-09-25T09_38_43_518Z-debug.log


```

**Comment by grillorafael** [comment]
HAving the same problem when try ing to update from `0.20.2` to `0.20.3`

**Comment by abernix** [comment]
Thanks for reporting this.  You should not be expected to install, or need, `wasm-pack`.  

I know what the problem is, I just need to put together for this, which I hope to have in a few hours.

Relates to #181

**Comment by abernix** [comment]
Should be fixed in `@apollo/gateway@0.20.4` which is now on the `latest` tag.  Please do report back if the problem is resolved or open a new issue if you find something new!  Thanks again!

**Comment by grillorafael** [comment]
Jus tried and it installed without problems
